src/Routes/RosSuggestedInstance/RosSuggestedInstance.js

```diff
@@ -15,11 +15,13 @@ import { useSelector, useDispatch } from 'react-redux';
 import { ServiceNotConfigured } from '../../Components/ServiceNotConfigured/ServiceNotConfigured';
 import { loadIsConfiguredInfo } from '../../store/actions';
 import NoEntitiesFound from '../../Components/NoEntitiesFound/NoEntitiesFound';
+import SuggestedInstanceTypesTable from  '../../Components/InstanceTypes/SuggestedInstanceTypesTable';
```

Comment:
Similar to this, could you please rename current directory and file name from Routes folder.
In case filename is too longer, in my opinion it is fine not to include `ROS` initial at the beginning of filename.  Thoughts?